### PR TITLE
chore(netlify) update build environnement

### PR DESCRIPTION
**Pull request by aslafy-z**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Use 
- NodeJS 10 LTS
- NPM 6.4.1

### CODE REVIEW

**Comment by aslafy-z** [comment]
@swapagarwal 

Could you please manualy clear the build cache for this failing build?  https://app.netlify.com/sites/peaceful-chandrasekhar-efde8e/deploys/5bffd9ed40e2a470ec83123d

Here the related doc: https://www.netlify.com/docs/build-gotchas/#build-cache

Thanks!

File: netlify.toml [hunk 1]
```diff
@@ -3,6 +3,10 @@
   publish   = "site/dist"
   command   = "gulp build"
 
+[build.environment]
+  NODE_VERSION = "11"
```
Comment (vikaspotluri123) [inline netlify.toml]: Netlify is a production environment --> should only use Node LTS

**Comment by swapagarwal** [comment]
@aslafy-z Done!

**Comment by swapagarwal** [comment]
Thanks @aslafy-z! 👍

**Pull request merged**